pkg/vcs/git.go

```diff
@@ -177,8 +177,8 @@ func (git *git) reset() error {
 		return nil
 	}
 	git.git("reset", "--hard", "--recurse-submodules")
-	git.git("clean", "-fdx")
-	git.git("submodule", "foreach", "--recursive", "git", "clean", "-fdx")
+	git.git("clean", "-xfdf")
```

Comment:
Does it really change the behavior? From [the manual](https://linux.die.net/man/1/git-clean) it looks like the flags are independent and their order doesn't matter.

Reply:
Flags are independent, but from the git [documentation](https://git-scm.com/docs/git-clean), a second -f flag will clean nested git repositories.

Reply:
I see, thanks!